packages/ra-core/src/controller/useEditController.ts

```diff
@@ -74,10 +70,6 @@ const useEditController = (props: EditProps): EditControllerProps => {
         },
     });
 
-    useEffect(() => {
-        dispatch(resetForm(REDUX_FORM_NAME));
```

Comment:
this was somehow necessary for the create to edit redirection...

Reply:
It's not necessary anymore